lang/julia/Portfile

```diff
@@ -92,6 +92,7 @@ post-destroot {
     move ${dpw}/julia-${version}/lib/libjulia.${short_version}.dylib ${destroot}${prefix}/lib
     move ${dpw}/julia-${version}/lib/libjulia.dylib ${destroot}${prefix}/lib
     move ${dpw}/julia-${version}/lib/libjulia.${major_version}.dylib ${destroot}${prefix}/lib
+    move ${dpw}/julia-${version}/libexec/julia ${destroot}${prefix}/libexec/julia
```

Comment:
Is this the critical line that resolves the issue?

Reply:
Yes.

There is `7z`, `dsymutil` and `lld` in `${destroot}${prefix}/libexec/julia`.  
ℹ️ `${destroot}${prefix}` being `/opt/local` by default.